build/charts/antrea/values.yaml

```diff
@@ -129,6 +131,11 @@ antreaProxy:
   # -- When set to false, AntreaProxy no longer load-balances traffic destined
   # to the External IPs of LoadBalancer Services.
   proxyLoadBalancerIPs: true
+  # -- The value of service.kubernetes.io/service-proxy-name label for AntreaProxy
+  # to match. If it is set, then AntreaProxy only handles the Service objects matching
+  # this label. The default value is empty string, which means that AntreaProxy will
```

Comment:
I checked the KEP, it says `When this label is set, kube-proxy will behave as if that service does not exist. None of the functionality that kube-proxy provides will be available for that service.`, so here you mean AntreaProxy will be only responsible to handle those Services with the label `service.kubernetes.io/service-proxy-name: 'antrea'` if serviceProxyName is configured like ` serviceProxyName: "antrea"`?

Reply:
Yes.